lib/charms/tls_certificates_interface/v1/tls_certificates.py

```diff
@@ -1348,7 +1402,7 @@ def _on_update_status(self, event: UpdateStatusEvent) -> None:
             time_difference = certificate_object.not_valid_after - datetime.utcnow()
             if time_difference.total_seconds() < 0:
                 logger.warning("Certificate is expired")
-                self.on.certificate_expired.emit(certificate=certificate)
+                self.on.certificate_invalidated.emit(reason="expired", certificate=certificate)
```

Comment:
Can we also pass chain, ca, csr to have the exact same context as in revoked?

Reply:
Done.